data-changes/location-codes.md

```diff
@@ -1,45 +1,55 @@
+Based on these nomenclature conventions:
+- **All GeoNet strong motion sensors have a location code `2?`**
+- **When a GeoNet station sensor is moved more than 1 m (but less than 200m), its location code will change accordingly (e.g. from `20` to `21`)**
 
-We are now retroactively correcting our data archive and our metadata records, so that this change is properly represented.
+### Change description
+After the convention was confirmed, we conducted an audit on all equipment changes across the GeoNet National Strong motion sensor network. 
+We found that a number of SM sensors were moved to a new location at the time without having their location code changed accordingly.
 
+We are now (2021) retroactively correcting our data archive and our metadata records, so that this change is properly represented.
 For those SM stations, location codes that were previously recorded as `20` are now being changed to `21`.
 
-Due to the complexity of this work, there has been for a period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. This has now been fixed.
+Due to the complexity of this work, there has been sometimes and  for a limited period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. 
+This has now been fixed.
 
 ### Affected data
 
 Data and metadata that are affected by this change are:
 - strong motion metadata (distributed via delta, FDSN)
 - strong motion miniseed data (distributed via FDSN)
 - strong motion processed data (distributed via the Strong Motion Tool and archive)
-- seismic data picks ID and magnitude ID (distributed via FDSN) might have metadata not correctly associated
+- seismic derived data: picks ID and magnitude ID (distributed via FDSN) might have metadata not correctly associated
 
-If you use strong motion miniseed data and metadata, you might be impacted by this change. The impact we expect can be of two forms:
-- if you have used and downloaded data between 2019 and 2021, you might have not been able to retrieve the complete dataset from FDSN
-- if you have a local copy of our data, things might not match anymore with our GeoNet archive
-- derived products we distribute and were not able to be reprocessed might still refer to old location code. For example, quakeML might still show that a station had location code 20, but then miniseed data will have that station with a location code 21
+If you use strong motion miniseed data and metadata, you might be impacted by this change.
+Impacts are expected:
+- if you used and downloaded data between 2019 and 2021, you might not be able to retrieve the exact same dataset from the Geonet FDSN data service
+ as it has been queried at that time
```

Comment:
shall we remove the phrase 'as it has been queried at that time'? I think the context is already captured even without this phrase

Reply:
changed by the betta 
all god